tests/fixture/main.ts

```diff
@@ -5,9 +5,15 @@
 /// <reference lib="deno.ns" />
 
 import { start } from "$fresh/server.ts";
+import { parseDenoFlags } from "../deps.ts";
 import routes from "./fresh.gen.ts";
 import options from "./options.ts";
 
-const experimentalDenoServe = Deno.args.includes("--experimental-deno-serve");
+const flags = parseDenoFlags(Deno.args, {
+  boolean: ["experimental-deno-serve"],
+});
```

Comment:
Moving to a proper parser for cli args is certainly a bit nicer, but this would've been better in a separate PR as they are not necessary to add TLS support. I know it's always tempting to add this other cleanup in a PR too, but that makes it harder for review the changes.